website/client/assets/scss/icon.scss

```diff
@@ -1,8 +1,6 @@
 .svg-icon {
   display: block;
   stroke-width: 0;
-  stroke: currentColor;
```

Comment:
those didn't do anything, thats why I moved them to 

```scss
&.color {
   svg path {
       stroke: currentColor;
       fill: currentColor;
     }
 }
```

but I can add them also back, if you want